### ISSUE TITLE
LGHT0091 : Duplicate symbol due to incorrect Directory entry in wxs file

### ISSUE THREAD

**Issue by greghmerrill** [felixrieseberg/electron-wix-msi#15]
To reproduce this problem, I created a new electronjs app using electron-forge (5.2.2):

```
electron-forge init gregtest2
```
The app runs fine, for example if I execute ```npm start```.  Next, I add a "wix" dependency to my package.json. 

```
  "config": {
    "forge": {
      "make_targets": {
        "win32": [
          "squirrel",
          "wix"
        ],
```

When this is present, electron-forge calls electron-wix-msi to generate an msi.  However, it ultimately fails, with the error below:

```
E:\views\dodc-ammo-2.1\electron\electron-forge\gregtest2\out\make\wix\x64\gregtest2.wxs(12808) : error LGHT0091 : Duplicate symbol 'Directory:resources_app_node_modules_yargs_parser_lib' found. This typically means that an Id is duplicated. Check to make sure all your identifiers of a given type (File, Component, Feature) are unique.
E:\views\dodc-ammo-2.1\electron\electron-forge\gregtest2\out\make\wix\x64\gregtest2.wxs(12897) : error LGHT0092 : Location of symbol related to previous error.
```

When I examine the wxs file created, I see an anomaly.  The Directory with Id ```resources_app_node_modules_yargs_parser_lib``` appears twice in the generated wxs file.  Once where it should, as a child of ```resources_app_node_modules_yargs_parser```:

```
                <Directory Id="resources_app_node_modules_yargs_parser" Name="yargs-parser">
                  <Directory Id="resources_app_node_modules_yargs_parser_lib" Name="lib">
                    <Component Id="resources_app_node_modules_yargs_parser_lib_tokenize_arg_string.js" Guid="48d2db54-3565-4cc6-9f55-f91c559be5d5">
                      <File Name="tokenize-arg-string.js" Id="resources_app_node_modules_yargs_parser_lib_tokenize_arg_string.js" Source="E:\views\dodc-ammo-2.1\electron\electron-forge\gregtest2\out\gregtest2-win32-x64\resources\app\node_modules\yargs-parser\lib\tokenize-arg-string.js" KeyPath="yes" Checksum="yes"/>
                    </Component>
                  </Directory>                  <Component Id="resources_app_node_modules_yargs_parser_CHANGELOG.md" Guid="dd26432b-0805-47ef-abe5-94c40700d7a8">
                    <File Name="CHANGELOG.md" Id="resources_app_node_modules_yargs_parser_CHANGELOG.md" Source="E:\views\dodc-ammo-2.1\electron\electron-forge\gregtest2\out\gregtest2-win32-x64\resources\app\node_modules\yargs-parser\CHANGELOG.md" KeyPath="yes" Checksum="yes"/>
                  </Component>
```

But the same Id appears again elsewhere, under Directory ```resources_app_node_modules_yargs```, which is NOT its parent:

```
                <Directory Id="resources_app_node_modules_yargs" Name="yargs">
                  <Directory Id="resources_app_node_modules_yargs_parser_lib" Name="lib">
                    <Component Id="resources_app_node_modules_yargs_lib_command.js" Guid="059cc3f5-3f8b-47f3-98a7-71265e4b6c80">
                      <File Name="command.js" Id="resources_app_node_modules_yargs_lib_command.js" Source="E:\views\dodc-ammo-2.1\electron\electron-forge\gregtest2\out\gregtest2-win32-x64\resources\app\node_modules\yargs\lib\command.js" KeyPath="yes" Checksum="yes"/>
                    </Component>
                    <Component Id="resources_app_node_modules_yargs_lib_completion.js" Guid="5beeb5f1-8c3f-4e2a-af1b-ebc3eff1d0c6">
                      <File Name="completion.js" Id="resources_app_node_modules_yargs_lib_completion.js" Source="E:\views\dodc-ammo-2.1\electron\electron-forge\gregtest2\out\gregtest2-win32-x64\resources\app\node_modules\yargs\lib\completion.js" KeyPath="yes" Checksum="yes"/>
                    </Component>
```

Here are the actual resources from under my app:
```
*wks-gmerrill* /cygdrive/e/views/dodc-ammo-2.1/electron/electron-forge/gregtest2/out/gregtest2-win32-x64
$ find resources/ -type d | grep yargs
resources/app/node_modules/yargs
resources/app/node_modules/yargs/lib
resources/app/node_modules/yargs/locales
resources/app/node_modules/yargs-parser
resources/app/node_modules/yargs-parser/lib
```

In summary, electron-wix-msi appears to think ```resources/app/node_modules/yargs-parser/lib``` is a child of both ```resources/app/node_modules/yargs-parser``` and ```resources/app/node_modules/yargs``` when it generates the wxs file.
